### PR TITLE
Updating submission process for connectors 

### PR DESCRIPTION
**Pull request by julie-campbell-sfdc**
Revised how to submit a connector based on:
- The new Contributor Console on the Exchange website
- The process for using GitHub to collect review materials (tfs [1373838](https://tfs.tsi.lan/tfs/DefaultCollection/Tableau-Dev/_workitems/edit/1373838))
- Input from the Partner Network about joining the program and go-to-market materials

### CODE REVIEW

**Review by pvanderknyff — COMMENTED** [review]
A couple of small nitpicks from a technical perspective

File: docs/gallery-submission.md [hunk 1]
```diff
@@ -140,82 +132,129 @@ To make things easy, we created templates for your custom installation instructi
 * [Tableau Prep Builder Installation Instructions JDBC Template](./templates/tableau-prep-jdbc-template.md)
 * [Tableau Server Installation Instructions Template](./templates/tableau-server.md)
 
-##### Icon
+#### Manifest file
```
[the remark this answers is not in the document]
Comment (julie-campbell-sfdc) [reply docs/gallery-submission.md:135]: I matched the name of these to how they are presented on the Exchange form. If the term "manifest" is confusing, should we request to change it on the Exchange itself? That field name is reused for accelerators and dashboard extensions, so I'm guessing they were just looking for a generic term.

For the purposes of this guide, is it okay to keep the header but change the body text, so contributors can identify the field on the form that we are referring to?
> Manifest file
> Upload the TACO file for your connector.

File: docs/gallery-submission.md [hunk 2]
```diff
@@ -17,116 +15,110 @@ title: Connector Tableau Exchange Submission Guide
...
-
-After you've built your integration and are passing 90% of TDVT results, sign up for our [Technology Partner Program](https://www.tableau.com/partners/resources/technology/partner-agreement). By signing up for the Technology Partner Program you allow us to discuss confidential information with you and enables us to promote your products through our marketing channels, like Tableau Exchange.
-
-### Step 4: Submit a Request to Publish
-
-After you've built and tested your integration, [submit a request](https://forms.gle/2FisXfmwwuQEaXKG9) to publish your connector in Tableau Exchange.
+### Step 3: Join the Tableau Partner Network
 
-### Step 5: Sign Tableau Exchange Agreement
+After you’ve built your integration and are passing 90% of TDVT results, join the Technology Track on our [Tableau Partner Network (TPN)](https://www.tableau.com/partners/become). By joining the TPN, you allow us to discuss confidential information with you and enable us to promote your products through our marketing channels, like the Tableau Exchange.
```
Comment (VasanthiNeelagiri) [inline docs/gallery-submission.md:55]: Can you also add this beside 90% of TDVT results- (all required tests should pass or acknowledged as not supported or known bug for the underlying data source).

File: docs/gallery-submission.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (VasanthiNeelagiri) [reply docs/gallery-submission.md:135]: I can request Exchange team to change the "manifest" field to ".taco"

File: docs/gallery-submission.md [hunk 3]
```diff
@@ -140,82 +132,129 @@ To make things easy, we created templates for your custom installation instructi
...
+- TDVT results csv file (test_results_combined.csv)
+- TDVT logs (tdvt_log_combined.txt, tabquery_logs.zip, and tdvt_actuals_combined.zip)
+- TDVT environment (INI config file)
 
----
-## **Beta Process**
+TDVT tests can be skipped for extract-only connectors.
 
-### Step 8: Release connector in Tableau Exchange for beta testing.
+In most cases, TDVT should be passing more than 90% of test cases to be accepted.
```
Comment (VasanthiNeelagiri) [inline docs/gallery-submission.md:195]: Can you also add this beside 90% of test cases to be accepted - All tests marked as required here - https://github.com/tableau/connector-plugin-sdk/blob/master/tdvt/tdvt/metadata/priority.csv
 should pass or acknowledged as not supported or known bug for the underlying data source
@pvanderknyff - Is there a better link to show which tests are required ?

**Review by VasanthiNeelagiri — APPROVED** [review]
Looks good to me. The only confusing thing is Manifest Vs Taco. I submitted a work intake request from them. We can change according to how they respond to it.

**Pull request merged**